### DISCUSSION TITLE
Equations not rendering properly

### DISCUSSION THREAD

**Discussion by Sbrjt** [darkreader/darkreader#10602] [General]
[page.zip](https://github.com/darkreader/darkreader/files/10385041/page.zip)

I was using this site that contained equations embedded as images (instead of latex) and turning on darkreader messed them up. Any fix?

<img src="https://user-images.githubusercontent.com/83166133/211615339-e1be7a7a-196e-4762-b6c8-b6c3a7df699a.png" height="200"> <img src="https://user-images.githubusercontent.com/83166133/211615391-c4d002d1-536e-470a-a77b-86090b8747e9.png" height="200">

(The site requires login so I have attached it as zip file)

**Comment by Gusted** [comment]
Hi, 

Please do not use the discussions tab to report issues, use [the "issues" tab](https://github.com/darkreader/darkreader/issues/new/choose) for that.

**Comment by gcb** [comment]
This page is still the top result for most terms related to this.

Most issues opened are fixed on a one-by-one basis, e.g. https://github.com/darkreader/darkreader/pull/1805
